clients/ts/FunctionalTests/package.json

```diff
@@ -6,17 +6,29 @@
   "main": "index.js",
   "dependencies": {
     "@aspnet/signalr": "file:../signalr",
-    "@aspnet/signalr-protocol-msgpack": "file:../signalr-protocol-msgpack"
+    "@aspnet/signalr-protocol-msgpack": "file:../signalr-protocol-msgpack",
+    "msgpack5": "^4.2.0"
   },
   "devDependencies": {
     "@types/debug": "0.0.30",
     "@types/jasmine": "^2.8.6",
+    "@types/karma": "^1.7.3",
     "@types/node": "^9.4.6",
     "debug": "^3.1.0",
     "es6-promise": "^4.2.2",
     "jasmine": "^3.1.0",
-    "tap-parser": "^7.0.0",
-    "tee": "^0.2.0",
+    "jasmine-core": "^3.1.0",
+    "karma": "^2.0.2",
```

Comment:
Do you need to tell someone about these?

Reply:
For test dependencies not really. We just need to do the needful.